docs/app/Examples/elements/Button/Types/ButtonIconExample.js

```diff
@@ -1,12 +1,10 @@
 import React, { Component } from 'react'
-import { Button, Icon } from 'stardust'
+import { Button } from 'stardust'
 
 export default class ButtonIconExample extends Component {
   render() {
     return (
-      <Button className='icon'>
-        <Icon className='world' />
-      </Button>
+      <Button icon='world' />
```

Comment:
Let's make sure we hit the Button API PR next so as to get these examples up and running as soon as possible. Since, right now they won't work with the current Button API.

I think it is OK to leave these updated though.